startpage_content.md

```diff
@@ -0,0 +1,62 @@
+This WIP PR is for preparing content for the start page.
+
+Please commit your ideas until 2019-05-11 so that we can discuss on the next web meeting about the design.
+
+Solutons could be:
+
+* screens (can take a lot of content)
```

Comment:
Not sure what you mean by "screens"... Screenshots?

Reply:
Screens in the sense of what Unpaywall (https://unpaywall.org/) is doing for instance (although this is not the most screen-esque website). The homepage is made of "slides" stacked vertically. The first one occupies the full screen.

It's mostly slides and a matter of organizing them (either in a carousel, with auto-sliding, or stacked vertically). :)